l10n/en/banners/m24-pencil-banner.ftl

```diff
@@ -4,4 +4,4 @@
 
 # Variables:
 # $donate (url) - link to https://foundation.mozilla.org/en/?form=donate-today
-m24-pencil-banner-support-ethical = Support ethical AI and internet privacy — <strong><a { $donate }>donate today!</a></strong>
```

Comment:
**Issue:** might we want to fallback to this older string after Dec 31st?

Reply:
✅  Official word is no. I added a switch like you suggested so we can disable the banner then.